README.md

```diff
@@ -6,16 +6,18 @@ This is the open-source portion of the back-end, website scraping software that
 
 ## Setup
 
-1.  Download chromium version 843427 locally.
-1.  Create a .env file with the following:
+1.  Download a recent version of Chromium locally: https://download-chromium.appspot.com/
+    - Optional: move to `/Applications` if you want:
```

Comment:
This file is currently platform independent.
Adding platform-specific information is probably unwise.
If it is necessary, it should be clearly labelled as such.

(That said, MacOS users who don't know how to move downloaded applications from Downloads/ to Applications/ are probably not the target audience for this github repo; although there are also probably plenty who would do it with the GUI rather than the command-line such that these this level of instruction just adds unnecessary confusion.)

Reply:
Good point. I've removed that line.

I've kept this example in the .env file to show that one should navigate into the dir all the way to `Chromium`, not just to `Chromium.app`:
`Applications/Chromium.app/Contents/MacOS/Chromium`